tsl/src/hypercore/hypercore_handler.c

```diff
@@ -82,6 +81,16 @@ static List *partially_compressed_relids = NIL; /* Relids that needs to have
 												 * updated status set at end of
 												 * transaction */
 
+static bool hypercore_truncate_compressed = true;
+
+bool
+hypercore_set_truncate_compressed(bool onoff)
+{
+	bool old_value = hypercore_truncate_compressed;
+	hypercore_truncate_compressed = onoff;
+	return old_value;
+}
```

Comment:
Maybe a comment about what this value and associated function does? Does it truncate the compressed table? Does it disable truncation of compressed tables? Under what circumstances would you not or would you not truncate compressed tables?

Reply:
Added.